modules/user-profile-migration/src/main/java/org/apache/airavata/utils/InstallCert.java

```diff
@@ -0,0 +1,192 @@
+package org.apache.airavata.utils;
```

Comment:
A couple questions about this file:
* Do we need this and if so what is it used for?
* Is the copyright notice on this file acceptable for inclusion in an Apache project?